src/app/destinyTrackerApi/d2-itemListBuilder.ts

```diff
@@ -8,83 +7,73 @@ import { D2ItemFetchRequest } from '../item-review/d2-dtr-api-types';
 import { translateToDtrItem } from './d2-itemTransformer';
 
 /**
- * Translates collections of DIM items into a collection of data almost ready to ship to the DTR API.
- * Generally tailored to work with weapon data.
- * Also tailored for the D2 version.
+ * Translate the universe of items that the user has in their stores into a collection of data that we can send the DTR API.
+ * Tailored to work alongside the bulkFetcher.
+ * Non-obvious bit: it attempts to optimize away from sending items that already exist in the ReviewDataCache.
```

Comment:
As a minor nit, this is (should be, unless I messed up) items that have a bulk fetch response already stored in the cache.

Reply:
You wrote it! I was just using the comment on the exported function.

Reply:
Whoever wrote the initial comment is a FOOL.